### DISCUSSION TITLE
sharp Attachment 질문

### DISCUSSION THREAD

**Discussion by Goodday777** [storycraft/node-kakao#515] [Q&A] [answered]
sharp Attachment 사용시 사용자간의 전달이 안되게끔 할수가 있나요?

기존 커스텀에서는 FW : false 와 같이 설정이 가능했는데.
샵에서도 그러한것이 있는지 궁금합니다.

**Comment by storycraft** [comment, marked as answer]
전달 여부 설정은 카카오링크 타입 메세지에서만 가능한것으로 보입니다. 추가로 현재 공식 클라이언트에서 사용하는 샵 검색 메세지는 카카오 링크와 같은 타입으로 바뀌었고, 기존 샵 검색 타입은 사용되지 않고 있기 때문에 언제든 작동하지 않을 수 도 있습니다.
